packages/rich-text-structure/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@wordpress/rich-text-structure",
+	"version": "1.0.0",
+	"description": "Rich text structure and manipulation API.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"rich-text"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/autop/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
```

Comment:
We should include the following entry point:
`"react-native": "src/index",`
as we will have to override all DOM related logic for RN.